builder/builder.go

```diff
@@ -738,14 +738,19 @@ func (b *Builder) AddBundles(bundles []string, allLocal bool, allUpstream bool,
 
 	// Add the ones passed in to the set
 	for _, bName := range bundles {
+		if _, exists := set[bName]; exists {
```

Comment:
optional: I prefer the `if _, ok := set[bName]; ok {` idiom represented in all the go docs.

Reply:
I thought the `ok` convention was used when the boolean returned represented success or failure, not existence. But I'm not purporting to be an expert at that at all.

Reply:
https://blog.golang.org/go-maps-in-action

Yeah, looks like you're right. That's.... um... huh. I guess that's a convention I don't agree with, but am willing to follow.

Reply:
Looks like I've thought this for a while and have several other places in the code I used "exists" like this. Hmm...

Reply:
Leaving this in for now 'cause that's how I did it everywhere else in the code. Can clean up later all together.